cargo-apk/src/apk.rs

```diff
@@ -231,7 +231,30 @@ impl<'a> ApkBuilder<'a> {
                 password: signing.keystore_password.clone(),
             },
             (None, true) => self.ndk.debug_key()?,
-            (None, false) => return Err(Error::MissingReleaseKey(profile_name.to_owned())),
+            (None, false) => {
```

Comment:
Tbh I don't mind supporting this for `dev` as well (after all, the documentation seems to imply that this should work OOTB), and I also think env vars should get precedence over config files - as is done with many other configurations like `RUSTFLAGS`? https://doc.rust-lang.org/cargo/reference/config.html#buildrustflags